pkg/events/queue/queue_mem_list.go

```diff
@@ -0,0 +1,114 @@
+package queue
+
+import (
+	"container/list"
+	"fmt"
+	"github.com/aquasecurity/tracee/pkg/external"
+	"sync"
+)
+
+type EventQueueMem struct {
```

Comment:
In continuation of above I think the `blocking` field should be defined here and initialized as pleased on queue construction.

Reply:
Same as previous comment. Thanks.